app/components/barcode_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%# Beware: This component is used in mailer content, so be mindful of email markup compatibility %>
+<%= content_tag(:figure, class: 'display-inline-block margin-0', **tag_options) do %>
+  <%= barcode_html.html_safe %>
+  <figcaption class="text-center margin-top-1">
+    <% if data_label.present? %>
+      <span class="usa-sr-only"><%= data_label %>:</span>
+    <% end %>
+    <%= formatted_data %>
+  </figcaption>
+<% end %>
```

Comment:
For [email compatibility](https://www.caniemail.com/features/html-semantics/), I may need to downgrade this markup. However, I might be able to emulate the same semantics with email-compatible markup using the [`figure` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/figure_role).

Reply:
Updated in 5d2d814